cl/simple_pages/urls.py

```diff
@@ -71,7 +71,13 @@
     path("terms/v/<int:v>/", old_terms, name="old_terms"),  # type: ignore[arg-type]
     path("terms/", latest_terms, name="terms"),  # type: ignore[arg-type]
     # Robots
-    path("robots.txt", robots, name="robots"),
+    path(
+        "robots.txt",
+        TemplateView.as_view(
+            template_name="robots.txt", content_type="text/plain"
+        ),
+        name="robots",
+    ),
```

Comment:
@mlissner is it ok to remove the cache decorator from this view? 🤔

Reply:
Yeah, should be fine. This view used to be dynamic, so it used to need a cache to perform well.